apps/files/src/components/FileFilterMenu.vue

```diff
@@ -1,50 +0,0 @@
-<template>
-  <v-menu class="mt-2" offset-y>
-    <v-btn slot="activator" flat @click="focusFilenameFilter"><v-icon large>filter_list</v-icon></v-btn>
-    <v-list>
-      <v-list-tile v-for="(filter, fid) in fileFilter" :key="fid">
-        <v-list-tile-title v-text="fileFilterTranslations[filter.name]"></v-list-tile-title>
-        <v-checkbox :input-value="filter.value" @change="setFileFilter({ name: filter.name, value: $event })"></v-checkbox>
-      </v-list-tile>
-      <v-list-tile>
-        <v-list-tile-title>
-          <span v-translate>Name Filter</span>
-        </v-list-tile-title>
-        <v-list-tile-title>
-        <search-bar @input="setFilterTerm" :value="filterTerm" ref="filenameFilter" autofocus :label="$gettext('Filter')" icon="false"/>
-        </v-list-tile-title>
-      </v-list-tile>
-    </v-list>
-  </v-menu>
-</template>
-
-<script>
-import { mapActions, mapGetters } from 'vuex'
-
-export default {
-  name: 'FileFilterMenu',
-  data () {
-    return {
-      fileFilterTranslations: {
```

Comment:
@LukasHirt this is where the filter texts are translated - please readd this

Reply:
Why is not the translation tag enough here? I thought that it works basically in the same way as gettext. I really need to go through the documentation of it 🙂

Reply:
the translation at runtime would work - but no the string extraction process.

a tool (gettext) is scanning the source code and extracts string e.g.
this.$gettext('Hello') -> Hello will be extracted
 <translate>Hello</translate> -> Hello will be extracted as well

 <translate>{{ filter.name }}</translate> -> there is no string to be extracted

Reply:
Ah. Okay, I see

Reply:
Fixed